### ISSUE TITLE
Explain Cross Entropy Loss

### ISSUE THREAD

**Issue by carschno** [carpentries-incubator/deep-learning-intro#545]
Episode 2 gives a high-level explanation of (Categorical) Cross Entropy Loss in section 5.
Perhaps add a box or a link (e.g. to [PyTorch documentation](https://pytorch.org/docs/stable/generated/torch.nn.CrossEntropyLoss.html)) that explains this in more depth and discusses alternative loss functions.
